controllers/configmap_controller.go

```diff
@@ -216,10 +230,50 @@ func (r *ConfigMapReconciler) reconcileServices(ctx context.Context, windowsServ
 			kubeTypes.NamespacedName{Namespace: r.watchNamespace, Name: windowsServices.Name}, "Error", err.Error())
 		return nil
 	}
+	r.setValues(node, data)
 	// TODO: actually react to changes to the services ConfigMap
 	return nil
 }
 
+func (r *ConfigMapReconciler) setValues(node *core.Node, data *servicescm.Data) {
+	// machineconfig.Spec.Config
+	// unmarshal the json, and set the data to the machineconfig's value
+
+	// check the node.annotations and node.label
+
+	for i := range data.Services {
+		value := &data.Services[i]
+		if value.Name == "kubelet" {
+			re := regexp.MustCompile(`(--node-labels=[^\s]+)`)
```

Comment:
This is duplicating logic from https://github.com/openshift/windows-machine-config-operator/blob/a66229cec4c125dc81eaa1b44e99db5812a51c42/pkg/services/services.go#L248
while also missing everything except node-labels.

Do both need to exist? If so the code should be maintained through shared functions.
It is not clear to me how that linked code works in tandem with this code.